cores/esp32/Esp.cpp

```diff
@@ -199,10 +204,10 @@ bool EspClass::flashEraseSector(uint32_t sector)
 
 bool EspClass::flashWrite(uint32_t offset, uint32_t *data, size_t size)
 {
-    return spi_flash_write(offset, (uint32_t*) data, size) == ESP_OK;
+    return spi_flash_write_encrypted(offset, (uint32_t*) data, size) == ESP_OK;
```

Comment:
Thanks for info. I'll revert this back and add note that this doesn't work with encrypted flash.